packages/nocodb/src/db/sql-mgr/code/models/xc/ModelXcMetaPg.ts

```diff
@@ -114,7 +114,7 @@ class ModelXcMetaPg extends BaseModelXcMeta {
         str = 'date';
         break;
       case 'double precision':
-        str = 'double';
+        str = 'float';
```

Comment:
This function is legacy and not used in recent versions, please revert this part as it is not required or applicable.